Source/Core/Core/DSP/DSPDisassembler.cpp

```diff
@@ -25,49 +25,14 @@ DSPDisassembler::DSPDisassembler(const AssemblerSettings& settings) : settings_(
 {
 }
 
-DSPDisassembler::~DSPDisassembler()
+bool DSPDisassembler::Disassemble(const std::vector<u16>& code, std::string& text)
```

Comment:
This only ever returns true now. Should this not check if the individual opcodes successfully disassemble and return false if one doesn't?